src/checks/incorrect_delimiter.rs

```diff
@@ -0,0 +1,88 @@
+use crate::checks::{Check, Warning};
+use crate::LineEntry;
+
+pub(crate) struct IncorrectDelimiterChecker {}
+
+impl Check for IncorrectDelimiterChecker {
+    fn run(&self, line: &LineEntry) -> Option<Warning> {
+        if line.raw_string.starts_with(' ') {
```

Comment:
I think you shoud not do this check because it's the work of another linter - https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/leading_space.rs.

Reply:
If this check is not included this linter returns false posititives for any `key` that mistakenly uses `" "` as a delimiter.

Reply:
I see. Maybe for that it's better to use method `is_whitespace()`?
```rust
if key.chars().any(|c| !c.is_alphabetic() && !c.is_whitespace() && c != '_') {}
```

Reply:
Ah good suggestion. Although, that would mean the linter would report `SOME KEY=true` as being correct. If this is acceptable behaviour I will action this change.

Reply:
Good catch! It's not acceptable behaviour. Can you add a test for that?

Reply:
But now this check returns some warnings if I have lines like those:
```env
ABCD= 1
ABCD =2
ABCD = 3
```
It's the work another linter - https://github.com/mgrachev/dotenv-linter/issues/9.
Could you fix it and cover with tests?